### DISCUSSION TITLE
MeshCommander: Allow disabling/disconnecting active/connected wireless device

### DISCUSSION THREAD

**Discussion by crass** [Ylianst/MeshCentral#4788] [General]
It is possible to use MeshCommander to disabling/disconnecting active/connected wireless device? I created a wireless profile that activated and connected the wireless device to an AP. Now I want to delete that wireless profile, but am unable to. I believe that AMT will not allow deleting of wireless profiles if they are associated with a connected wireless device. How can I do this currently? Or do I need to make a feature request?

**Comment by jsastriawan** [comment]
Try disable wifi on AMT and then remove the profile.

**Reply by crass** [reply]
When you say "on AMT", do you mean via the BIOS text mode MEBx interface?

**Reply by jsastriawan** [reply]
You can use meshcommander or AMT webui at HTTP port 16992/ HTTPS port 16993. If you LMS installed, you can access it via localhost from the host itself.

**Reply by crass** [reply]
Ok, still confused. I want to use MeshCommander to disable the wifi, without LMS installed. Where is the ability to do that in the MeshCommander UI? 

Here's what I see under Network for the wireless device:
![AMT-Wireless-state](https://user-images.githubusercontent.com/740863/202565130-d1aa5661-ee48-4079-b0a4-aae5ca642ae7.png)

Where else might it be?

**Reply by jsastriawan** [reply]
See "State" row. Click on the blue link. It should show other possible value.

**Reply by crass** [reply]
That was it, thanks! In retrospect, that was obvious.
